### ISSUE TITLE
Option to trim floating points

### ISSUE THREAD

**Issue by h0adp0re** [narugit/tmux-temp-mac#3]
Hey, I have a request for an option to trim floating points from the temperature as I don't need the reading to be that precise.

**Comment by narugit** [comment]
@R0LA1mRifcF8yAkk 

Sounds great.

So, you want to get not `45.8℃` but `46℃` in the option mode, right?

**Comment by h0adp0re** [comment]
Yeah, I hadn't thought of rounding the numbers but that makes a lot of sense. So yeah, exactly like that.
